_pages/travel-and-leave/travel-and-leave-policies/first-time-travel-complete-concur-profile.md

```diff
@@ -19,7 +19,7 @@ You will want to complete your Concur profile to pay for hotels listed in Concur
 
 It will make booking hotels in Concur easier, and allow you to personally redeem miles and/or points earned while on government travel.
 
-1. Log into Concur [travel.gsa.gov](https://travel.gsa.gov/).
+1. On VPN, log into Concur [travel.gsa.gov](https://travel.gsa.gov/). (Concur does work off VPN as well, but you'll have to go through the password reset process for access.)
```

Comment:
Does a user have to go through the "password reset" process every time they log into Concur off the VPN, or only the first time? (I ask because I've never logged onto the VPN to use Concur, and I've never had to use the password reset thing. I'm curious if I got lucky or you got unlucky, and in the latter case, we should probably fix whatever is happening. 😛 )

Reply:
I don't know! Didn't have time to get that far.